spec/jobs/reindex_model_job_spec.rb

```diff
@@ -14,11 +29,10 @@
     Apartment::Tenant.switch!(account.tenant) { work }
   end
 
-  it "#perform_later can enqueue model for reindex later" do
-    expect { described_class.perform_later(work.class.to_s, account.cname) }.to have_enqueued_job(described_class).exactly(:once)
-  end
-
-  it "#perform_now will enqueue model for reindex" do
-    expect { described_class.perform_now(work.class.to_s, account.cname) }.to have_enqueued_job(described_class).exactly(:once)
+  it "sets doi_status_when_public to findable" do
+    expect(work.doi).to be_empty
+    described_class.perform_now(work.class.to_s, account.cname, limit: 1, options: options)
+    work.reload
+    expect(work.doi_status_when_public).to eq "findable"
```

Comment:
You can combine this into one line, as it is standard in job tests to have to reload the object changed by the job.
````
expect(work.reload.doi_status_when_public).to eq "findable"
````